src/model/CreateBitcoinEstimateRequest.js

```diff
@@ -19,7 +19,7 @@ class CreateBitcoinEstimateRequest {
       obj = obj || new CreateBitcoinEstimateRequest();
 
       if (data.hasOwnProperty('timestamp')) {
-        obj['timestamp'] = ApiClient.convertToType(data['timestamp'], 'String');
+        obj['timestamp'] = ApiClient.convertToType(data['timestamp'], 'Date');
```

Comment:
Technically breaking, but will bet that no one uses it yet. 

One more reason to be careful with the swagger definitions!